framework/synapse/pipeline/create_lake_db.json

```diff
@@ -40,7 +40,7 @@
 						}
 					},
 					"sparkPool": {
-						"referenceName": "spark3p2sm",
+						"referenceName": "spark3p2med",
```

Comment:
I don't think we should change the node size unless "spark3p2med" is necessary for executing the pipeline.